tools/utils/array_readwriter/__init__.py

```diff
@@ -1,2 +1,2 @@
-from . import csv, numpy_array
+from . import csv, numpy_array, parquet
```

Comment:
`array_readwriter` exists both under `//tools/utils/` and `//tools/distpartitioning`. this looks weird.
And in this file, it use `from . import csv, numpy_array` which indicates import csv from current directory. but actually not.

Reply:
forget to remove these 2 files and directory?

Reply:
Nice catch, let me delete the redundant directory.